src/modeler/AutoRest.Swagger.Tests/SwaggerModelerValidationTests.cs

```diff
@@ -594,6 +594,13 @@ public void PutResponseResourceValidationTest()
         }
 
         [Fact]
+        public void LROStatusCodesValidationTest()
```

Comment:
Ran linter rules against the whole repo, a cursory look at the logs did not reveal any false positives. It does flag some cases where it does make sense to not have a response model for 201/200 status. Eg: `Apis_CreateOrUpdate` or `FileSystem_RemoveDefaultAcl`